customer-service/src/main/java/com/spaghettininjas/yaposs/controller/CustomerController.java

```diff
@@ -0,0 +1,37 @@
+package com.spaghettininjas.yaposs.controller;
+
+import com.spaghettininjas.yaposs.service.CustomersService;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import com.spaghettininjas.yaposs.repository.entity.Customer;
+import java.util.Arrays;
+
+@RestController
+@RequestMapping("/api/Customers")
+public class CustomerController {
+
+    @Autowired
+    private CustomersService service;
+
+    @RequestMapping(value = "/{id}", method = RequestMethod.GET, produces=MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<Customer> get(@PathVariable int id) {
+        return service.findById((long) id)
+                .map(customer -> ResponseEntity.ok().body(customer))
+                .orElse(ResponseEntity.notFound().build());
+    }
+
+    @RequestMapping(value = "/{id}", method = RequestMethod.DELETE, produces = MediaType.APPLICATION_JSON_VALUE)
+    public void deleteById(@PathVariable int id){
+        service.deleteById((long) id);
+    }
+
+    @PostMapping
```

Comment:
I don't how `save` works exactly, so I might be wrong, but this looks like it functions like both update and create. Maybe [`existsById`](https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/repository/CrudRepository.html#existsById(ID)) can be used to to differentiate the two, or maybe change the api to only have `upsert` instead of separate  end-points for creation and update

Reply:
For the sake of interest, you can try, but this is not necessary

Reply:
addressed in https://github.com/domanteli0/PoSS/pull/4/commits/84ca7b3bbdd41a6b79b83808cfd2337adfb52d07